Tests/scripts/cp_gcp_dir.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+gsutil -m cp -r "$1" "$2"
```

Comment:
Seem like a waste to have a file for a one line exec. You can simply call subprocess to execute this command with the specified args.

Reply:
I originally used it as part of the python code but was told to move it to a bash script. So should I put it as part of the python code anyway?

Reply:
It can be done from the config yml

Reply:
Yes it can be done from the config yml, however it means moving some logic from python to config yml. What is our preference?

Reply:
You can do this in Python with one line 
```python
subprocess.check_output(['gsutil', '-m', 'cp', '-r, src, dest])
```
Best to remove this file and have python call subprocess

Reply:
Sure will do so.